README.md

```
@@ -1,21 +1,108 @@
 # fantasy-premier-league
 
+### Project Codebase
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)